packages/lumx-react/src/components/popover/Popover.tsx

```diff
@@ -155,7 +155,7 @@ const _InnerPopover = forwardRef<PopoverProps, HTMLDivElement>((props, ref) => {
     const clickAwayRefs = useRef([popoverRef, anchorRef]);
     const mergedRefs = useMergeRefs<HTMLDivElement>(setPopperElement, ref, popoverRef);
 
-    return isOpen
+    return isOpen && styles.popover
```

Comment:
Looks like this might be a problem in unit tests as it will defer the display of the popover...